### ISSUE TITLE
Error when trying to use get_ncaa_baseball_roster

### ISSUE THREAD

**Issue by TravisRPetersen** [BillPetti/baseballr#184]
Getting an error when trying to run get_ncaa_baseball_roster. Error message is below.

I'm not an R expert but when I try to use pd.read_html with the url https://stats.ncaa.org/team/528/roster/14781, I get a 403 error so maybe that is the same reason?

/opt/miniconda3/lib/python3.7/urllib/request.py in http_error_default(self, req, fp, code, msg, hdrs)
    647 class HTTPDefaultErrorHandler(BaseHandler):
    648     def http_error_default(self, req, fp, code, msg, hdrs):
--> 649         raise HTTPError(req.full_url, code, msg, hdrs, fp)
    650 
    651 class HTTPRedirectHandler(BaseHandler):

HTTPError: HTTP Error 403: Forbidden

```
Error: Can't combine `..1$url_slug` <logical> and `..3$url_slug` <character>.
Traceback:

1. get_ncaa_baseball_roster(teamid = 528, year = 2019)
2. lapply(payload, function(x) parse_roster_table(x)) %>% dplyr::bind_rows()   # at line 65-66 of file <text>
3. dplyr::bind_rows(.)
4. vec_rbind(!!!dots, .names_to = .id)
5. (function () 
 . vec_default_ptype2(x = x, y = y, x_arg = x_arg, y_arg = y_arg, 
 .     `vctrs:::from_dispatch` = `vctrs:::from_dispatch`, `vctrs:::df_fallback` = `vctrs:::df_fallback`, 
 .     `vctrs:::s3_fallback` = `vctrs:::s3_fallback`))()
6. vec_default_ptype2(x = x, y = y, x_arg = x_arg, y_arg = y_arg, 
 .     `vctrs:::from_dispatch` = `vctrs:::from_dispatch`, `vctrs:::df_fallback` = `vctrs:::df_fallback`, 
 .     `vctrs:::s3_fallback` = `vctrs:::s3_fallback`)
7. stop_incompatible_type(x, y, x_arg = x_arg, y_arg = y_arg, `vctrs:::from_dispatch` = match_from_dispatch(...))
8. stop_incompatible(x, y, x_arg = x_arg, y_arg = y_arg, details = details, 
 .     ..., message = message, class = c(class, "vctrs_error_incompatible_type"))
9. stop_vctrs(message, class = c(class, "vctrs_error_incompatible"), 
 .     x = x, y = y, details = details, ...)
10. abort(message, class = c(class, "vctrs_error"), ...)
11. signal_abort(cnd)
```

**Comment by BillPetti** [comment]
Thanks, this was fixed by #183

**Comment by TravisRPetersen** [comment]
Thanks!
